package.json

```diff
@@ -1,55 +1,101 @@
 {
   "name": "contentful-batch-libs",
-  "version": "0.0.0-determined-by-semantic-release",
+  "version": "11.0.0-beta.1",
   "description": "Library modules used by contentful batch utility CLI tools.",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/types/index.d.ts",
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.cjs"
+    },
+    "./dist/contentful-batch-libs.node.cjs": "./dist/contentful-batch-libs.node.cjs",
+    "./dist/contentful-batch-libs.browser.js": "./dist/contentful-batch-libs.browser.js",
+    "./dist/contentful-batch-libs.react-native.js": "./dist/contentful-batch-libs.react-native.js",
+    "./test/mocks/index.js": "./test/mocks/index.js"
+  },
+  "main": "./dist/cjs/index.cjs",
+  "types": "dist/types/index.d.ts",
   "engines": {
-    "node": ">=18"
+    "node": ">=20"
   },
   "scripts": {
-    "build": "npm run clean && tsc",
-    "build:watch": "tsc --watch",
+    "build": "npm run clean && npm run build:esm && npm run build:rollup && rm -rf dist/esm-raw",
+    "build:esm": "tsc",
+    "build:rollup": "NODE_ENV=production rollup -c",
+    "build:docs": "typedoc",
+    "postbuild": "npm run check",
+    "check": "npm run check:cjs && npm run check:esm && npm run check:browser && npm run check:browser-min",
+    "check:browser": "es-check es2018 ./dist/contentful-batch-libs.browser.js",
```

Comment:
these files don't exist in the build so it is currently failing. See my comment in the rollup file about the naming of them